README.md

```diff
@@ -3,12 +3,16 @@
 
 Bazel rules for generating TypeScript declarations for JavaScript protocol buffers using the 
 [ts-protoc-gen](https://github.com/improbable-eng/ts-protoc-gen) protoc plugin. These rules can also
-generate service definitions for use by [grpc-web](https://github.com/improbable-eng/grpc-web).
+generate service definitions for use by [grpc-web](https://github.com/improbable-eng/grpc-web) or
+[grpc-node](https://github.com/grpc/grpc-node).
 
 ## Getting Started
 
 > If you're migrating from the ts-protoc-gen rules, see [here](docs/migrating_from_ts_protoc_gen.md) for a migration guide
```

Comment:
You can delete this line and replace it with the new one, it's been over a year now so this probably isn't relevant anymore.